### PR TITLE
BaSP-TG-38: list super admin

### PR DESCRIPTION
**Pull request by CarVigil**

### CODE REVIEW

File: src/Components/SuperAdmins/index.js [hunk 1]
```diff
@@ -1,9 +1,43 @@
+import { useEffect, useState } from 'react';
+import dotenv from 'dotenv';
```
Comment (Gearyandres94) [inline src/Components/SuperAdmins/index.js:2]: It shouldnt be necessary to import dotenv in the file.

File: src/Components/SuperAdmins/index.js [hunk 2]
```diff
@@ -1,9 +1,43 @@
...
 import styles from './super-admins.module.css';
+import List from './List/List';
+
+dotenv.config();
```
Comment (Gearyandres94) [inline src/Components/SuperAdmins/index.js:6]: This should also be removed.

Comment (Gearyandres94) [inline src/Components/SuperAdmins/index.js:6]: This should also be removed.

**Review by GuidoCerioni — APPROVED** [review]
Nice work!

**Review by Facund0Fernandez — APPROVED** [review]

**Review by franco-marini — COMMENTED** [review]
Just left you a little comment, everything else looks good to me

File: src/Components/SuperAdmins/ListItem/ListItem.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      deleteSuperAdmin(listItem._id);
+    }
+  };
+  return (
+    <tr>
+      <td>{listItem.firstName}</td>
+      <td>{listItem.lastName}</td>
+      <td>{listItem.email}</td>
+      <td>
+        <button className={styles.btn} onClick={() => handleDelete(listItem._id)}>
```
[the remark this answers is not in the document]
Comment (CarVigil) [reply src/Components/SuperAdmins/ListItem/ListItem.js]: Honestly, I don't know why I put it in that moment. I delete the parametern and works fine.

**Review by HiginiaMed — APPROVED** [review]

**Review by HiginiaMed — APPROVED** [review]
Good Job!

**Pull request merged**